clients/web/src/lib/components/atoms/Modal/Modal.story.svelte

```diff
@@ -8,7 +8,7 @@
 </script>
 
 <Hst.Story title="Atoms/Modal" layout={{type: "grid", width: 800}}>
-	<Hst.Variant title="Default" {open}>
+	<Hst.Variant title="Default">
```

Comment:
Why remove this? Won't this prevent the story controls from working?

Reply:
There aren't any controls for this story; functionality still works.
This was creating type errors in vscode, hence the removal